app/jobs/check_veteran_residence_location_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class CheckVeteranResidenceLocationJob < CaseflowJob
+  RESIDENCE_LOCATION_PROCESS_LIMIT = if ENV["RESIDENCE_LOCATION_BATCH_SIZE"].nil?
+                                       2500
+                                     else
+                                       ENV["RESIDENCE_LOCATION_BATCH_SIZE"].to_i
+    end
```

Comment:
```suggestion
  RESIDENCE_LOCATION_PROCESS_LIMIT = ENV.fetch("RESIDENCE_LOCATION_BATCH_SIZE", 2500).to_i
```

[ENV.fetch](https://ruby-doc.org/core-2.7.2/ENV.html#method-c-fetch) allows for specifying a default value.